src/resources/projects/components/ProfessionalsSection/index.tsx

```diff
@@ -1,26 +1,165 @@
-import { FC } from 'react';
-import { ProjectProfessionalResponse } from 'types';
+import {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useCallback,
+  useMemo,
+  useState,
+} from 'react';
+import { ProfessionalResponse, ProjectProfessionalResponse } from 'types';
 import Professional from './Professional';
-import { Box } from '@mui/material';
+import {
+  Box,
+  Button,
+  DialogContentText,
+  Autocomplete,
+  AutocompleteRenderInputParams,
+  TextField,
+} from '@mui/material';
 import { SectionTitle } from 'components/ui';
+import { useRedirect, required, TextInput } from 'react-admin';
+import Dialog from 'components/Dialog';
+import { addSource } from 'utils';
 
 type ProfessionalsSectionProps = {
   professionals: ProjectProfessionalResponse[] | undefined;
+  allProfessionals?: ProfessionalResponse[] | undefined;
+  setProjectProfessionals?: Dispatch<
+    SetStateAction<ProjectProfessionalResponse[]>
+  >;
+  isEdit: boolean;
```

Comment:
```suggestion
  isEdit?: boolean;
```
Make it optional, so we dont need to set up if it is false